crates/sncast/src/helpers/rpc.rs

```diff
@@ -57,27 +55,13 @@ impl RpcArgs {
     }
 }
 
-fn installation_constant_seed() -> Result<u64> {
-    let executable_path = current_exe()?;
-    let metadata = executable_path.metadata()?;
-    let modified_time = metadata.modified()?;
-    let duration = modified_time.duration_since(UNIX_EPOCH)?;
-
-    Ok(duration.as_secs())
-}
-
 enum FreeProvider {
     Blast,
-    Voyager,
 }
 
 impl FreeProvider {
     fn semi_random() -> Self {
-        let seed = installation_constant_seed().unwrap_or(2);
-        if seed % 2 == 0 {
-            return Self::Blast;
-        }
-        Self::Voyager
+        FreeProvider::Blast
     }
```

Comment:
It's called `semi_random` and it always returns blast - I know that this PR is a hotfix but this should be changed at some point, either by adding other provider or refactoring the code so the method name is not confusing. Could you please link an issue here?

Reply:
I'd want to introduce an alternative provider in the future, there exist some other. But don't have time to research them right now.

Reply:
And generally rework this logic, so provider can be overriden by the user. Will do that post hotfix

Reply:
Yes sure, let's just not forget about it 👍